### DISCUSSION TITLE
Some HGT files missing from the server

### DISCUSSION THREAD

**Discussion by Sublimis** [mapsforge/mapsforge#1589] [General]
Up there. Could you have a look at that, please?
(For example the file named N76E065.hgt.zip, it's not present)

![image](https://github.com/user-attachments/assets/a4fbd776-3fef-42ef-8873-a305b49c0b19)

**Comment by devemux86** [comment]
@Sublimis 

> For example the file named N65E065.hgt.zip, it's not present

[N65E065.hgt.zip](https://download.mapsforge.org/maps/dem/dem3/N65/N65E065.hgt.zip) exists on the server.
Do you mean something else?

These terrain data tiles are taken from [viewfinderpanoramas.org](http://viewfinderpanoramas.org/) and are repackaged.

**Reply by Sublimis** [reply]
Sorry, a typo, should be N76E065.hgt.zip.

**Comment by devemux86** [comment]
The files were originally repackaged [here](https://bailu.ch/dem3/) some years ago.

There may be new dem3 files from viewfinderpanoramas that could be included:
https://viewfinderpanoramas.org/Coverage%20map%20viewfinderpanoramas_org3.htm

We need the repackaged files in the same zip format like the existing ones
and we can include them in the download server.

**Reply by Sublimis** [reply]
There are definitely some HGT files in viewfinderpanoramas.org DEM3 data that are not present here, e.g. contents of https://viewfinderpanoramas.org/dem3/T41.zip
